modal/cli/environment.py

```diff
@@ -39,10 +40,25 @@ def __rich__(self):
 def list(json: Optional[bool] = False):
     envs = environments.list_environments()
     table_data = []
+
+    env_var_default = environ.get("MODAL_ENVIRONMENT") or None
```

Comment:
But actually — do we need to separately check `os.envion`? That's already handled by our `Config.get`: https://github.com/modal-labs/modal-client/blob/main/modal/config.py#L233-L250

Reply:
yup! I took it out